amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/disposition/DispositionServiceImpl.java

```diff
@@ -539,7 +535,7 @@ private boolean hasDisposableItemsImpl(boolean isRecordLevelDisposition, NodeRef
             }
             else if (filePlanService.isRecordCategory(item) && getAssociatedDispositionScheduleImpl(item) == null)
             {
-                if (hasDisposableItemsImpl(isRecordLevelDisposition, item));
+                if (hasDisposableItemsImpl(isRecordLevelDisposition, item))
                 {
```

Comment:
Can we combine both the if condition together or create a different method for evaluating the whole condition

Reply:
done the changes as per the comment.

Reply:
Please correct code indentation.

Reply:
Its proper we no need to do changes.